packages/core/src/toast/Toast.tsx

```diff
@@ -30,12 +30,18 @@ export const Toast = forwardRef<HTMLDivElement, ToastProps>(function Toast(
 
   return (
     <div
-      className={clsx(withBaseName(), withBaseName(status), className)}
+      className={clsx(
+        withBaseName(),
+        withBaseName(status ?? "default"),
```

Comment:
Can this be implemented in the same way as Tooltip please

Reply:
made it as per tooltip